modules/git/blame.go

```diff
@@ -154,6 +158,11 @@ func CreateBlameReader(ctx context.Context, objectFormat ObjectFormat, repoPath
 	cmd.AddDynamicArguments(commit.ID.String()).AddDashesAndList(file)
 
 	done := make(chan error, 1)
+	// Use errPipe to preserve the original err and avoid overwriting it.
```

Comment:
Are you serious? Why there is still this wrong comment?

Reply:
This is the fourth time that I spend time on telling you about the "named return value" problem. Even though I have shown you the full demo code before:

https://github.com/go-gitea/gitea/pull/34149#discussion_r2032457935



![image](https://github.com/user-attachments/assets/41b5a594-84dc-40b4-a7be-033041795949)

Reply:
I think this thread is about the err variable names not the same problem as you mentioned.

Reply:
A new name is to reduce confusing not because of overriding. My comment was not right.